docs/contributor/04-20-release.md

```diff
@@ -0,0 +1 @@
+# Kyma Environment Broker release pipeline
```

Comment:
There's no content in this doc. Is this intended?

Reply:
Yes, it will be in the next PR to not make this one too big.